From 28bf7a8ee0820f5e501d5aee55aa77c4c2463bf2 Mon Sep 17 00:00:00 2001
From: svenate <salini.venate@intel.com>
Date: Mon, 31 Jan 2022 10:24:39 +0530
Subject: [PATCH] Update security_patch_level string

Security_patch_level needs to be updated when
ASB patches are integrated.

Tracked-On: NA
Signed-off-by: svenate <salini.venate@intel.com>
---
 core/version_defaults.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/version_defaults.mk b/core/version_defaults.mk
index 47bb92c142..f8c511de1d 100644
--- a/core/version_defaults.mk
+++ b/core/version_defaults.mk
@@ -240,7 +240,7 @@ ifndef PLATFORM_SECURITY_PATCH
     #  It must be of the form "YYYY-MM-DD" on production devices.
     #  It must match one of the Android Security Patch Level strings of the Public Security Bulletins.
     #  If there is no $PLATFORM_SECURITY_PATCH set, keep it empty.
-      PLATFORM_SECURITY_PATCH := 2022-01-05
+      PLATFORM_SECURITY_PATCH := 2022-02-01
 endif
 .KATI_READONLY := PLATFORM_SECURITY_PATCH
 
-- 
2.17.1

